wordpress-mobile/WordPress-iOS
# [Notifications P1] Add missing notification selection color for a split screen

Fixes #22672

This PR fixes the lack of the notification rows' selection style on iPad (during split notifications mode). I also modified the rows' paddings to make the selection look natural. iPad multitasking mode is also handled to switch the selection style accordingly to the screen's width.

| Before | After |
| -------| ------|
|![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-02-22 at 16 08 40](https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/043fad63-11a3-4a7f-9ea6-a08a36f377c9)|![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-02-22 at 16 12 09](https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/c88a3eaf-7d17-4da0-9ca6-adfc5474b410)|

<details>
  <summary>Selection with no padding changes</summary>
  
![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-02-22 at 16 18 38](https://github.com/wordpress-mobile/WordPress-iOS/assets/16563318/e55d7c5d-6f82-4fe9-a8dd-3c10e6158e35)
  
</details>

To test:
- Run the app on an iPad
- Go to Notifications and ensure that a selected notification row has a grey-ish background
- Run the app on an iPhone
- Go to Notifications and make sure that by tapping on a notification, you can't see any row's background changes

## Regression Notes
1. Potential unintended areas of impact
Notifications screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests, UI tests

3. What automated tests I added (or what prevented me from doing so)
UI tests already exist

PR submission checklist:


Testing checklist:
